README.md

```diff
@@ -23,334 +23,25 @@ Please refer to the [CHANGELOG](CHANGELOG.md) for version history and known issu
 
 ## Installation
 
-This gem ships as part of Chef Workstation. See the [Chef Workstation Download Page](https://downloads.chef.io/products/workstation) to download the installation package.
+The kitchen-google driver ships as part of Chef Workstation. The easiest way to use this driver is to [Download Chef Workstation](https://www.chef.io/downloads/tools/workstation).
```

Comment:
`kitchen-google` should be `kitchen-vcenter`

Reply:
Fixed